tests/test_devices.py

```diff
@@ -115,9 +115,9 @@ def test_without_status(self, alarm_device):
         """Verify that querying items without specifying a status works correctly"""
         alarm_device.connect("username", "password")
         sectors = {
-            0: {"element": 1, "excluded": False, "id": 1, "index": 0, "name": "S1 Living Room", "status": True},
-            1: {"element": 2, "excluded": False, "id": 2, "index": 1, "name": "S2 Bedroom", "status": True},
-            2: {"element": 3, "excluded": False, "id": 3, "index": 2, "name": "S3 Outdoor", "status": False},
+            0: {"element": 1, "activable": True, "id": 1, "index": 0, "name": "S1 Living Room", "status": True},
+            1: {"element": 2, "activable": True, "id": 2, "index": 1, "name": "S2 Bedroom", "status": True},
+            2: {"element": 3, "activable": False, "id": 3, "index": 2, "name": "S3 Outdoor", "status": False},
```

Comment:
I'm not sure about this, as this change should add a behavior and not change the behavior.